src/polymer-language-server.ts

```diff
@@ -87,7 +87,7 @@ connection.onInitialize((params): InitializeResult => {
     urlLoader: new FSUrlLoader(workspacePath),
     urlResolver: new PackageUrlResolver(), polymerJsonPath
   });
-  documents.all().forEach(d => scanDocument(d));
+  mapAllInitialDocuments(editorService);

```

Comment:
That's right. The containing function isn't async. Really we should wait until after returning. This too is handled much better in #62